app/components/message_attachments/action_button/action_button_text.js

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Text, View} from 'react-native';
+import Emoji from 'app/components/emoji';
+import {getEmoticonName} from 'app/utils/emoji_utils';
+
+// reEmoji matches an emoji (eg. :taco:) at the start of a string.
+const reEmoji = /^:([a-z0-9_\-+]+):\B/i;
+
+// reEmoticon matches an emoticon (eg. :D) at the start of a string.
+const reEmoticon = /^(?:(:-?\))|(;-?\))|(:o)|(:-o)|(:-?])|(:-?d)|(x-d)|(:-?p)|(:-?[[@])|(:-?\()|(:['’]-?\()|(:-?\/)|(:-?s)|(:-?\|)|(:-?\$)|(:-x)|(<3|&lt;3)|(<\/3|&lt;\/3)|(:[`'’]-?\(|:&#x27;\(|:&#39;\())(?=$|\s|[*_~?])/i;
+
+// reMain matches some amount of plain text, starting at the beginning of the string and hopefully stopping right
+// before the next emoji by looking for any character that could start an emoji (:, ;, x, or <)
+const reMain = /^[\s\S]+?(?=[:;x<]|$)/i;
+
+const containerStyles = {
+    alignItems: 'flex-start',
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+};
+
+export function ActionButtonText({message, style}) {
```

Comment:
Can you change this so that it's the default export to be consistent with other components as well?

Reply:
Done.